docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]
+`
+
+**DESCRIPTION**  
+It keeps the copy of tarball on the pbench-agent and pushes the other to pbench-server
+
+**OPTIONS**  
+--user  
+This option value is required if not provided by the
+'PBENCH_USER' environment variable; otherwise, the value provided
+on the command line will override any value provided by the
+environment.
+
+--controller  
+This option may be used to override the value
+provided by the 'PBENCH_CONTROLLER' environment variable; if
+neither value is available, the result of 'hostname -f' is used.
+(If no value is available, the command will exit with an error.)
+
+--prefix  
+This option allows the user to specify an optional
+directory-path hierarchy to be used when displaying the result
+tar balls on the pbench server.
+
+--show-server  
+This will not move any results, but resolve and
+then display the pbench server destination for results.
+
+--xz-single-threaded  
+This will force the use of a single
+thread for locally compressing the result tar balls.
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-list-tools
+
+---
+
+**NAME**  
+pbench-list-tools, List all the registered tools with group, host, and remote info.
+
+**SYNOPSIS**  
+`pbench-list-tools [-C][-n][-g][-o][--help]
+`
+
+**DESCRIPTION**  
+List all tools that are registered and can filter by name of the group
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-n, --name STR  
+list the tool groups in which <tool-name> is used.
+
+-g, --group STR  
+list the tools used in this <group-name>
+
+-o, --with-option  
+list the options with each tool
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-list-triggers
+
+---
+
+**NAME**  
+pbench-list-triggers, Lists the registered triggers by group
+
+**SYNOPSIS**  
+`pbench-list-triggers[-C][-g][--help]
+`
+
+**DESCRIPTION**  
+This command will list all the registered triggers by group-name
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-g, --group,--groups STR  
+List all the triggers registered in the <group-name> group.
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-move-result
+
+---
+
+**NAME**  
+pbench-move-results, Move all results to pbench-server
+
+**SYNOPSIS**  
+`pbench-move-results [--help] [--user=<user>] [--controller=<controller>] [--prefix=<path>] [--xz-single-threaded] [--show-server]
+`
+
+**DESCRIPTION**  
+Move result directories to the configured pbench-server. Once the tarball is moved successfully, it clears the local copy of the tarball in the pbench-agent.This command is used when result tarball is pushed via SSH
```

Comment:
"used when [...] pushed via SSH" is something, but, more specifically, this command can only be used for a 0.69 Pbench Server destination, and requires a `/opt/pbench-agent/id_rsa` file with the private SSH key of the server's `pbench` account.